src/v2/Apps/Settings/Routes/DeleteAccount/DeleteAccountRoute.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button, Checkbox, Separator, Text, TextArea, useToasts
+} from "@artsy/palette"
+import * as React from "react"
+import { useState } from "react"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { deleteMyAccount } from "../../Mutations/DeleteMyAccount"
+import { useSystemContext } from "v2/System/SystemContext"
+
+export const DeleteAccountRoute = () => {
+  const { relayEnvironment } = useSystemContext()
+
+  const [isConfirmed, setConfirmed] = useState(false)
+  const [enteredReason, setEnteredReason] = useState("")
+  const isDisabled = !isConfirmed || !enteredReason
+  const { sendToast } = useToasts()
+
+  const handleDeleteMyAccount = async () => {
+    const url = window.location.href
+    try{
+      await deleteMyAccount(relayEnvironment!, enteredReason, url)
+      window.location.href = "/"
+      sendToast({
+        variant: "success",
+        message: "Profile deleted successfully",
+      })
+    } catch (error) {
+      const parsedError = JSON.parse(error.message)
```

Comment:
Why is this returning unparsed JSON?

Reply:
I'm not sure! Maybe we missed something in the mutation function?